cmd/snap/cmd_prepare_image.go

```diff
@@ -171,6 +171,12 @@ func (x *cmdPrepareImage) Execute(args []string) error {
 	opts.AppArmorKernelFeaturesDir = x.AppArmorKernelFeaturesDir
 	opts.SysfsOverlay = x.SysfsOverlay
 
+	dir, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+	opts.SeedManifestDir = dir
```

Comment:
before prepare-image itself was not writing this, so my preference would be to add a new option that takes optionally a filepath:

--write-revisions  => writes ./seed.manifest
--write-revisions=filepath => writes to the given filepath
no --write-revisions, old behavior, no writing

Reply:
Ah OK! I was under the impression we wanted to write this unconditionally! I'll implement this switch, and add a 'documentation needed' tag to this